profiles/webrecorder.js

```diff
@@ -0,0 +1,75 @@
+import * as ReSpec from "../src/respec.js";
+
+const modules = [
+  // order is significant
+  import("../src/core/location-hash.js"),
+  import("../src/core/l10n.js"),
+  import("../src/webrecorder/defaults.js"),
+  import("../src/core/style.js"),
+  import("../src/webrecorder/style.js"),
+  import("../src/core/github.js"),
+  import("../src/core/data-include.js"),
+  import("../src/core/markdown.js"),
+  import("../src/core/reindent.js"),
+  import("../src/core/title.js"),
+  import("../src/webrecorder/headers.js"),
+  import("../src/webrecorder/abstract.js"),
+  import("../src/core/data-transform.js"),
+  import("../src/core/data-abbr.js"),
+  import("../src/core/inlines.js"),
+  import("../src/webrecorder/conformance.js"),
+  import("../src/core/dfn.js"),
+  import("../src/core/pluralize.js"),
+  import("../src/core/examples.js"),
+  import("../src/core/issues-notes.js"),
+  import("../src/core/best-practices.js"),
+  import("../src/core/figures.js"),
+  import("../src/core/webidl.js"),
+  import("../src/core/biblio.js"),
+  import("../src/core/link-to-dfn.js"),
+  import("../src/core/xref.js"),
```

Comment:
You might not need `xref`, `ui/search-xref` modules, unless you reference definitions and terms from W3C specs. This should reduce bundle size and avoid unintended dependency on W3C related stuff.

Also potentially:
- `mdn-annotation`
- `webidl`
- `webidl-index`
- `linter-rules/wpt-tests-exist`.